packages/types/src/config.ts

```diff
@@ -115,6 +115,8 @@ export type ClientConfig = {
     FileLevel: string;
     FeatureFlagBoardsProduct: string;
     FeatureFlagGraphQL: string;
+    FeatureFlagAnnualSubscription: string;
+    FeatureFlagPlanUpgradeButtonText: string;
```

Comment:
Is `FeatureFlagPlanUpgradeButtonText` supposed to be added? I think maybe it was accidentally included when resolving merge conflicts, because it was deliberately removed in this [PR](https://github.com/mattermost/mattermost-webapp/pull/11442/files)

Reply:
Oh I thought that was newly added but maybe not. I'll remove that